lib/rubocop_director/commands/generate_config.rb

```diff
@@ -8,6 +8,11 @@ class GenerateConfig
 
       RUBOCOP_TODO = ".rubocop_todo.yml"
 
+      def initialize(director_config:, todo_config:)
+        @director_config = director_config
+        @todo_config = todo_config
```

Comment:
is it possible for `rubocop --regenerate-todo` to use the override?

Reply:
Probably not. I didn't find anything. but what if someone has already moved the todo file to some other path and wants to use it instead?

Reply:
I'd rather not care about this scenario, we should not fight with tool we use 🙂

Reply:
got it. I'll remove this functionality